tests/NLog.UnitTests/LogFactoryTests.cs

```diff
@@ -38,39 +38,49 @@ namespace NLog.UnitTests
     using System.IO;
     using System.Threading;
     using NLog.Config;
-    using NLog.UnitTests.Mocks;
     using Xunit;
 
     public class LogFactoryTests : NLogTestBase
     {
         [Fact]
         public void Flush_DoNotThrowExceptionsAndTimeout_DoesNotThrow()
         {
-            using (new NoThrowNLogExceptions())
-            {
-                LogManager.ThrowExceptions = true;
+            // Arrange
+            LogManager.Configuration = XmlLoggingConfiguration.CreateFromXmlString($@"
+            <nlog throwExceptions='false'>
+                <targets>
+                    <target type='BufferingWrapper' name='test'>
+                        <target type='MethodCall' name='test_wrapped' methodName='{nameof(TestClass.GenerateTimeout)}' className='{typeof(TestClass).AssemblyQualifiedName}' />
+                    </target>
+                </targets>
+                <rules>
+                    <logger name='*' minlevel='Debug' writeto='test'></logger>
+                </rules>
+            </nlog>");
 
-                LogManager.Configuration = XmlLoggingConfiguration.CreateFromXmlString(@"
-                <nlog throwExceptions='false'>
-                    <targets><target type='MethodCall' name='test' methodName='Throws' className='NLog.UnitTests.LogFactoryTests, NLog.UnitTests.netfx40' /></targets>
-                    <rules>
-                        <logger name='*' minlevel='Debug' writeto='test'></logger>
-                    </rules>
-                </nlog>");
+            Logger logger = LogManager.GetCurrentClassLogger();
+            logger.Info("Prepare Timeout");
 
-                ILogger logger = LogManager.GetCurrentClassLogger();
-                logger.Factory.Flush(_ => { }, TimeSpan.FromMilliseconds(1));
-            }
+            Exception timeout = null;
```

Comment:
please rename this variable, it's a strange name for an exception ;)

Reply:
Fixed